community-spotlight-conference-backend-world-affairs-conference.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Modernizing the conference management backend of the World Affairs Conference'
+description: 'Putting the spotlight on Samuel Martineau, a software engineer based out of Ontario, Canada replacing the backend of a conference management system with Xata.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cs-wac-cover.jpg
+  alt: Community Spotlight
+author: Alex Francoeur
+date: 05-08-2024
+tags: ['community']
+published: true
+slug: community-spotlight-conference-backend-world-affairs-conference
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cs-wac-cover.jpg
+---
+
+[Samuel Martineau](https://github.com/Samuel-Martineau) has not even graduated high school yet and is already racking up an impressive list of professional accomplishments. From being on national TV for his [wall climbing robot](https://github.com/Samuel-Martineau/memomath), to leading the team and architectural design for the new management system of the worlds oldest student run conference, the [World Affairs Conference](https://worldaffairscon.org/). The conference brings in over 4,000 of the worlds brightest minds from 25 countries in 65 schools internationally. As Samuel gets ready to graduate from the [Upper Canada College](https://en.wikipedia.org/wiki/Upper_Canada_College), he looks to take his talents to the world of mechatronics, pursuing a degree in mechanical engineering at a Canadian university.
+
+## Modernizing a dated tech stack
+
+Samuel was tasked with choosing a tech stack for the World Affairs Conference that was meant to last the next decade. While there’s been some changes to the tech stack since the inception of the conference in the 80’s, they were overdue for an upgrade.
+
+![World Affairs Conference landing page](/images/cs-wac-website.png)
+
+For the backend, there were a few very specific requirements for different parts of the registration flows, administrative dashboards and general budget. The backend needed to:
+
+- Be able to host images
+- Speak SQL and support relational data models
+- Be a fully managed and hosted solution
+- Be cost effective and fall within the constraints of the conference budget
+
+Xata checked all of the boxes and more for their next generation tech stack.
+
+> I think we used pretty much every feature Xata offers. Our user profile and speaker images are stored in Xata, we used the summarize API for sign up and registration time series charts, preview databases with Netlify, full text search for our application and more. Having one service to provide all of this functionality made it a pretty easy decision to choose Xata.\
```

Comment:
We currently do not use the full text search functionality, though we do expect to use it in the near future.

Reply:
Updated ✅